grouper/repositories/permission_grant.py

```diff
@@ -24,13 +25,19 @@
 
 if TYPE_CHECKING:
     from grouper.graph import GroupGraph
+    from grouper.entities.pagination import Pagination
     from grouper.models.base.session import Session
     from typing import Dict, Iterable, List, Optional
 
 
 class GraphPermissionGrantRepository(PermissionGrantRepository):
     """Graph-aware storage layer for permission grants."""
 
+    SORT_FIELD = {
+        PermissionGrantSortKey.GROUP: "group",
+        PermissionGrantSortKey.SERVICE_ACCOUNT: "service_account",
+    }
```

Comment:
Hmm, not sure how I feel about using the sort keys in this way. I think we could instead have only a single sort key ("name" probably) and simplify things that way (e.g. removes the need for this dict at all). There are separate methods below anyways, so it's not like you actually need the sort key to distinguish between group grants vs. service account grants.

Reply:
Changing it to name involves changing the data class and lots of other places, which is a little annoying. I refactored to avoid the dict. Let me know if that's unsatisfactory